set.mm

```diff
@@ -536981,6 +536982,29 @@ standard proof of the implication in modal logic (B5 ` => ` 4).  Its dual
   bj-wnfenf $p |- ( ( E. x ph -> A. x ps ) -> A. x ( E. x ph -> ps ) ) $=
     ( wex wal wi bj-wnf1 bj-19.21bit sylg ) ACDZBCEFZKJBFCABCGJBCHI $.
 
+  $( Equivalent form of the axiom of substitution ~ bj-ax12 .  Although both
+     sides need a DV condition on ` x , t ` (or as in ~ bj-ax12v3 on
+     ` t , ph ` ) to hold, their equivalence holds without DV conditions.  The
+     forward implication is proved in modal (K4) while the reverse implication
+     is proved in modal (T5).  The LHS has the advantage of not involving
+     nested quantifiers on the same variable.  Note that in the LHS, the
+     reverse implication holds by ~ equs4 (or ~ equs4v if a DV condition is
+     added on ` x , t ` as in ~ bj-ax12 ).
+
+     The LHS can be read as saying that if there exists a setvar equal to a
+     given term witnessing ` ph ` , then all setvars equal to that term also
+     witness ` ph ` .  An equivalent suggestive form for the LHS is
+     ` -. ( E. x ( x = t /\ ph ) /\ E. x ( x = t /\ -. ph ) ) ` , which
+     expresses that there can be no two variables both equal to a given term,
+     one witnessing ` ph ` and the other witnessing ` -. ph ` .  (Contributed
+     by BJ, 21-May-2024.)  (Proof modification is discouraged. $)
```

Comment:
We should probably ensure that ~bj-ax12 is not used?

Reply:
Statements in a mathbox are by default not used outside of that mathbox (one has to add the option `/INCLUDE_MATHBOXES` to `MINIMIZE_WITH <label>` in order to use them). And in my mathbox, I'm happy with using it later.  Can I keep it that way ?

Reply:
Yes, fine for me!